e2e-tests/mdx/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "gatsby-mdx-tester",
```

Unless there's something specific about the monorepo setup that I don't know about, this should probably be marked as *private*, right?
```suggestion
  "name": "gatsby-mdx-tester",
  "private": true,
```

---

I was pattern matching with the other e2e directories and none of them are private: true. May be a miss rather than intentional though.

---

Definitely a miss - all those e2e tests should have `private: true`, but in fairness - those are not part of workspaces/lerna so they are not versioned/published anyway